app/carbonapi/app.go

```diff
@@ -529,7 +550,10 @@ func deferredAccessLogging(r *http.Request, accessLogDetails *carbonapipb.Access
 		accessLogger.Info("request served", zap.Any("data", *accessLogDetails))
 		apiMetrics.Responses.Add(1)
 	}
-	prometheusMetrics.Responses.WithLabelValues(fmt.Sprintf("%d", accessLogDetails.HttpCode), accessLogDetails.Handler).Inc()
+
+	if app != nil {
+		app.prometheusMetrics.Responses.WithLabelValues(fmt.Sprintf("%d", accessLogDetails.HttpCode), accessLogDetails.Handler).Inc()
```

Comment:
Here is better way to use strconv.Itoa. See https://gist.github.com/evalphobia/caee1602969a640a4530